### ISSUE TITLE
it needs TCP now on upcoming 18.2 

### ISSUE THREAD

**Issue by frootloop88** [nythepegasus/SideJITServer#161]
but I guess this project is dead anyway

**Comment by nythepegasus** [comment]
Not dead, merely dormant. I do mean to return to this project and update it, I've just been in a billion different places.

**Comment by Gammaxana** [comment]
Hello, pooping in here to add that I have gotten this working on ios 18.2 Public beta and with a 16 pro max, try changing the Port associated to this with the -p to 2002 and adjust accordingly the side jit shortcut and you should be good to go

**Comment by frootloop88** [comment]
> Hello, popping in here to add that I have gotten this working on ios 18.2 Public beta and with a 16 pro max, try changing the Port associated to this with the -p to 2002 and adjust accordingly the side jit shortcut and you should be good to go

That’s great news but can you explain that in more
Details please?

**Comment by frootloop88** [comment]
> Ah right, so I was getting errors and it wasn't working with the default port, so I changed the port flag -p 2002 which is a port I know I'm not using. It made it to where I saw the TCP established and it worked after that. It also took changing the IP on the shortcut to be http://[my IP]:2002 . Once that was done it loaded right up.

where do I put the -p flag? on sudo SideJITServer --pair ?

**Comment by nythepegasus** [comment]
@frootloop88 Correct, `-p` is `--port`, `--pair` is `-y` since `-p` was for port.

**Comment by frootloop88** [comment]
> @frootloop88 Correct, `-p` is `--port`, `--pair` is `-y` since `-p` was for port.

so I do sudo SideJITSERVER --pair -p 2002 ?

**Comment by Gammaxana** [comment]
> > @frootloop88 Correct, `-p` is `--port`, `--pair` is `-y` since `-p` was for port.
> 
> 
> 
> so I do sudo SideJITServer --pair -p 2002 ?

That looks right, go ahead and give it a go and report back

**Comment by frootloop88** [comment]
> > > @frootloop88 Correct, `-p` is `--port`, `--pair` is `-y` since `-p` was for port.
> > 
> > 
> > so I do sudo SideJITServer --pair -p 2002 ?
> 
> That looks right, go ahead and give it a go and report back

it doesn't. of course I changed the ip and port in shortcuts too. Still getting   File "/Users/macintosh/venv/lib/python3.9/site-packages/pymobiledevice3/remote/tunnel_service.py", line 441, in start_quic_tunnel
    raise QuicProtocolNotSupportedError(
pymobiledevice3.exceptions.QuicProtocolNotSupportedError: iOS 18.2+ removed QUIC protocol support. Use TCP instead (requires python3.13+)
